lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -342,7 +342,7 @@ def refine_lines_gaussian(
                 initial_sigma=initial_sigma,
                 fixed_background=fixed_background,
             )
-            tmp_positions.append(r.x[1] / image._pixel_size)
+            tmp_positions.append(r[0][1] / image._pixel_size)
```

Comment:
why is only the first peak stored?

Reply:
Because this is feature parity and only the one peak case (see PR description).